src/api/index.js

```diff
@@ -17,3 +17,4 @@ export { default as subscriptionAdd } from './subscriptionAdd';
 export { default as subscriptionRemove } from './subscriptionRemove';
 export { default as sendMessage } from './sendMessage';
 export { default as uploadFile } from './uploadFile';
+export { default as getRawMessage } from './getRawMessage';
```

Comment:
lets go with `getSingleRawMessage` ;-)
why its important I think - because generally message is returned as html here we'd like to get content only.